kubectl-plugin/pkg/cmd/version/version.go

```diff
@@ -0,0 +1,20 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var Version = "development"
+
+func NewVersionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Show kubectl ray version",
```

Comment:
nit: "Output the version of the Ray kubectl plugin"

Reply:
done